include/OutboundEmail/OutboundEmail.php

```diff
@@ -199,40 +199,56 @@ function getUserMailers($user) {
         		      $autoCreateUserSystemOverride = TRUE;
         		 }
 
-                //Substitute in the users system override if its available.
-                if($userSystemOverride != null)
-        		   $system = $userSystemOverride;
-        		else if ($autoCreateUserSystemOverride)
-        	       $system = $this->createUserSystemOverrideAccount($user->id,"","");
-
-			    $isEditable = ($system->type == 'system') ? FALSE : TRUE; //User overrides can be edited.
-
-                if( !empty($system->mail_smtpserver) )
-				    $ret[] = array('id' =>$system->id, 'name' => "$system->name", 'mail_smtpserver' => $system->mail_smtpdisplay,
-								   'is_editable' => $isEditable, 'type' => $system->type, 'errors' => $systemErrors);
+                // Substitute in the users system override if its available.
+                if($userSystemOverride != null) {
+                    $system = $userSystemOverride;
+                }
+        		else if ($autoCreateUserSystemOverride) {
+                    $system = $this->createUserSystemOverrideAccount($user->id, "", "");
+                }
+                // User overrides can be edited.
+                $isEditable = ($system->type == 'system' || $system->type == 'system-override') ? FALSE : TRUE;
+
+                if( !empty($system->mail_smtpserver) ) {
+                    $ret[] = array(
+                        'id' => $system->id,
+                        'name' => "$system->name",
+                        'mail_smtpserver' => $system->mail_smtpdisplay,
+                        'is_editable' => $isEditable,
+                        'type' => $system->type,
+                        'errors' => $systemErrors
+                    );
+                }
 			}
 			else //Sendmail
```

Comment:
PSR-2:
"else and elseif are on the same line as the closing brace from the earlier body."